### ISSUE TITLE
Garden `0.13.15` `garden dev` crashes on start.

### ISSUE THREAD

**Issue by mattpolzin** [garden-io/garden#5146]
## Crash report

When I run `garden dev` (which was functioning for `0.13.14`) I now get a crash.

### Error message

```
 🌸  Initializing...


/snapshot/pkg/core/node_modules/ink/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js:53
        throw ex;
        ^

TypeError: fsevents.watch is not a function
    at createFSEventsInstance (/snapshot/pkg/core/node_modules/chokidar/lib/fsevents-handler.js:93:25)
    at setFSEventsListener (/snapshot/pkg/core/node_modules/chokidar/lib/fsevents-handler.js:149:16)
    at FsEventsHandler._watchWithFsEvents (/snapshot/pkg/core/node_modules/chokidar/lib/fsevents-handler.js:351:18)
    at FsEventsHandler.initWatch (/snapshot/pkg/core/node_modules/chokidar/lib/fsevents-handler.js:427:23)
    at FsEventsHandler._addToFsEvents (/snapshot/pkg/core/node_modules/chokidar/lib/fsevents-handler.js:516:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.5.0
```

### What did you do?

`garden dev` (or any variant of using `sync` mode).

### Your environment

* **OS:** macOS 13.5.1 (22G90) -- Intel Mac
* **How I'm running Kubernetes:** Azure
* **Garden version:** 0.13.15

### Frequency

Happens every time consistently.

### Workaround

Downgrade to version `0.13.14`.

**Comment by shumailxyz** [comment]
Thanks for reporting this. Just tried with a fresh install of 0.13.15 on intel & apple silicon mac and couldn't reproduce it. 

But i wonder if this could be due to https://github.com/garden-io/garden/pull/5133.

**Comment by mattpolzin** [comment]
Hmm, frustrating that it wasn't immediately reproducible. My co-worker also saw the same crash on his Apple Silicon laptop, so that's maybe an interesting data point.

I can try a fresh install just to see.

**Comment by mattpolzin** [comment]
I did get the same crash with a fresh install of Garden.

**Comment by twelvemo** [comment]
I am able to reproduce this on an Intel Mac. We are working on a fix right away @mattpolzin Thanks for reporting!

**Comment by shumailxyz** [comment]
@mattpolzin We released the fix for this issue in 0.13.16. Please give it a try. 
https://github.com/garden-io/garden/releases/tag/0.13.16

Thanks again for reporting the issue.

**Comment by mattpolzin** [comment]
Can confirm that fixed it. Thanks!
